modular_skyrat/modules/nullf­a­g_masspr/README.txt

```diff
@@ -0,0 +1,16 @@
+NullF­a­g here
+i'm just gonna say outright, this method of modularisation is an extremely a­u­t­i­s­t­i­c and r­e­t­a­r­d­e­d one, and i'm only obliging
+by it because *a particular maintainer* would lose their non-existent mind otherwise
```

Comment:
ah yes, insulting the maintainers will definitely put you in a good light

Reply:
Wasn't there literally an announcement recently about not attacking devs?